packages/jest-cli/src/TestRunner.js

```diff
@@ -379,6 +379,10 @@ class TestRunner {
   _setupReporters() {
     const config = this._config;
 
+    if (config.reporters) {
+      return this._addCustomReporters(config.reporters);
```

Comment:
although in some cases that might be a desired behavior. I'm not really sure how to go about that. maybe add another option `noDefaultReporters: true`?